pkg/plugin/plugin.go

```diff
@@ -229,7 +274,13 @@ func New(binPath string, pluginID string) (*Plugin, error) {
 	}
 
 	//nolint:exhaustruct
-	plgn := &Plugin{status: Starting, BinPath: binPath + exe, ID: pluginID, quitChan: make(chan bool)}
+	plgn := &Plugin{
+		status:   Starting,
+		BinPath:  binPath + exe,
+		ID:       pluginID,
+		quitChan: make(chan bool),
+	}
+	plgn.info, _ = plgn.getInformation()
```

Comment:
you should not ignore the error

```suggestion
	plgn.info, err = plgn.getInformation()
	if err != nil {
		return nil, fmt.Errorf("getting plugin information: %w", err)
	}
```